### ISSUE TITLE
Depth Anything V2

### ISSUE THREAD

**Issue by emanuelevivoli** [LiheYoung/Depth-Anything#191]
Link broken?

**Comment by LiheYoung** [comment]
Yes. Strangely, our DepthAnything organization account and the Depth-Anything-V2 homepage account have been flagged by Github. They are hidden from the public now. We are appealing to recover them. Sorry for the inconvenience. Our online demo is still alive: https://huggingface.co/spaces/depth-anything/Depth-Anything-V2.

**Comment by ArdavanSh** [comment]
in the huggingface, it seems there is no configuration json file?

**Comment by LiheYoung** [comment]
Hi @juntaosun, we are undergoing company review procedures to release the Depth-Anything-Giant checkpoint. It is temporarily not available. You can try our Depth-Anything-Large. It has similar performance as the giant model.

**Comment by LiheYoung** [comment]
Hi @emanuelevivoli, thank you for your interest. Due to the issue of our Github repositories, we upload our original GitHub repository content to Huggingface space now: https://huggingface.co/spaces/depth-anything/Depth-Anything-V2/tree/main

**Comment by LiheYoung** [comment]
Hi @ArdavanSh, we provide our raw PyTorch checkpoints in the model card, without the need of configuration json file. You can use them following the instructions here: https://huggingface.co/spaces/depth-anything/Depth-Anything-V2/blob/main/README_Github.md.

**Comment by LiheYoung** [comment]
Hi @juntaosun, Depth-Anything-V2-Small model is under the Apache-2.0 license. Depth-Anything-V2-Base/Large/Giant models are under the CC-BY-NC-4.0 license.  See this: https://huggingface.co/spaces/depth-anything/Depth-Anything-V2/blob/main/README_Github.md#license

**Comment by ArdavanSh** [comment]
Thank you for the response @LiheYoung. Are we able to use the Depth-Anything-V2-Metric-VKITTI-Large model the same way?

**Comment by LiheYoung** [comment]
Yes. The instructions for using our metric depth models are here: https://huggingface.co/spaces/depth-anything/Depth-Anything-V2/tree/main/metric_depth. The two metric depth models are under the Apache-2.0 license.

**Comment by XiaoqiangZhou** [comment]
Awesome research work! Thanks for your contribution.

**Comment by joreeves** [comment]
Great job! Will you be releasing the relative depth training code?

**Comment by spacewalk01** [comment]
Thank for your great work. I integrated to depth-anything-v2 version to tensorrt
https://github.com/spacewalk01/depth-anything-tensorrt

**Comment by PoopBear1** [comment]
Thank you for excellent work!
Do you have any idea that how long would the link be restored?
at this moment, I still cannot clone the V2 repo:
```
git clone https://github.com/DepthAnything/Depth-Anything-V2
Cloning into 'Depth-Anything-V2'...
remote: Repository not found.
fatal: repository 'https://github.com/DepthAnything/Depth-Anything-V2/' not found
```

**Comment by spacewalk01** [comment]
@PoopBear1 you can clone V2 version from huggingface as follows: 
`git clone https://huggingface.co/spaces/depth-anything/Depth-Anything-V2/tree/main/depth_anything_v2`

**Comment by LiheYoung** [comment]
Our V2 repository and project page have been recovered: https://github.com/DepthAnything/Depth-Anything-V2, https://depth-anything-v2.github.io. Sorry for the inconvenience to you.

**Comment by YacineDeghaies** [comment]
> Great job! Will you be releasing the relative depth training code?

Where you able to train using relative depth ?
